src/test/java/com/amazon/dlic/auth/ldap/LdapBackendTest.java

```diff
@@ -354,7 +354,7 @@ public void testLdapAuthorization() throws Exception {
                 .put(ConfigConstants.LDAP_AUTHZ_ROLEBASE, "ou=groups,o=TEST")
                 .put(ConfigConstants.LDAP_AUTHZ_ROLENAME, "cn")
                 .put(ConfigConstants.LDAP_AUTHZ_ROLESEARCH, "(uniqueMember={0})")
-                // .put("opendistro_security.authentication.authorization.ldap.userrolename",
```

Comment:
Can all the code commented out be removed?

Reply:
IMO, It should be.

Reply:
Any concerns of removing these?

Reply:
Let's not remove this since its not related to this PR